entc/gen/graph.go

```diff
@@ -643,17 +644,25 @@ func (a assets) write() error {
 
 // format runs "goimports" on all assets.
 func (a assets) format() error {
-	for _, file := range a.files {
-		path := file.path
-		src, err := imports.Process(path, file.content, nil)
-		if err != nil {
-			return fmt.Errorf("format file %s: %v", path, err)
-		}
-		if err := ioutil.WriteFile(path, src, 0644); err != nil {
-			return fmt.Errorf("write file %s: %v", path, err)
-		}
+	var eg errgroup.Group
+
+	for i := range a.files {
+		idx := i
+		f := a.files[idx]
+		eg.Go(func() error {
+			path := f.path
+			src, err := imports.Process(path, f.content, nil)
+			if err != nil {
+				return fmt.Errorf("format file %s: %v", path, err)
+			}
+			if err := ioutil.WriteFile(path, src, 0644); err != nil {
+				return fmt.Errorf("write file %s: %v", path, err)
```

Comment:
similar to above, a more clear message might be "issue writing file '%v': %w", ...